### ISSUE TITLE
'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. 

### ISSUE THREAD

**Issue by JacobMarley** [mljar/mljar-supervised#498]
I'm getting a bunch of these in a venv after installing requirements.txt. Any tips on how to fix?

'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.
'verbose_eval' argument is deprecated and will be removed in a future release of LightGBM. Pass 'log_evaluation()' callback via 'callbacks' argument instead.
'evals_result' argument is deprecated and will be removed in a future release of LightGBM. Pass 'record_evaluation()' callback via 'callbacks' argument instead.

**Comment by pplonski** [comment]
Looks like we need to update LightGBM api that is used in AutoML. Thanks @JacobMarley for reporting!

**Comment by pplonski** [comment]
@MaciekEO this is similar to #514 - with additional warning:
```
'verbose_eval' argument is deprecated and will be removed in a future release of LightGBM. Pass 'log_evaluation()' callback via 'callbacks' argument instead.
```

**Comment by pplonski** [comment]
Fixed in #514
